app/services/controlled_work_document_population_service.rb

```diff
@@ -29,6 +29,7 @@ def call(session_data, model)
       "cw1_and_2" => "lib/cw1-and-2-form-2023-8-21.pdf",
       "civ_means_7" => "lib/civ-means-7-form.pdf",
       "cw1_welsh" => "lib/cw1-form-welsh.pdf",
+      "cw1_welsh_new" => "lib/cw1-form-welsh-new.pdf",
```

Comment:
Should we call this just cw1_welsh and the old one cw1_welsh_legacy?

Reply:
same as other PR, using legacy means all forms need to be renamed, so we will just add the _new suffix to the new forms

Reply:
Oh yes, that makes sense. 
Should we then date `cw1_welsh_new` & `cw1_new` like `cw1_welsh_sep24` & `cw1_sep24` ?

Reply:
I think _new is sufficient, it only needs to differentiate between 2 forms for a limited time and it will encourage us to revert back to the original naming convention once all forms are complete

Reply:
Make sense 👍🏽